projects/repast/src/validate.js

```diff
@@ -0,0 +1,119 @@
+import {
+  always,
+  append,
+  curry,
+  dropLast,
+  ifElse,
+  keys,
+  last,
+  length,
+  map,
+  nth,
+  pipe,
+  propEq,
+  reduce,
+  reject,
+  isNil,
+  anyPass,
+  __,
+  any,
+} from "ramda";
+import { print } from "recast";
+import F from "fluture";
+import vm from "vm";
+
+import { PRIMITIVE_TYPES } from "./constants";
+import {
+  isUnionType,
+  parseUnionType,
+  isCompositeType,
+  parseCompositeType,
+} from "./types";
+
+const parseSignature = pipe(
+  reduce(
+    (parsed, symbol) => (symbol === "->" ? parsed : append(symbol, parsed)),
+    []
+  ),
+  (s) => ({ params: dropLast(1, s), returnType: last(s) })
+);
+
+const typeToValue = (type) => {
+  if (type === "String") {
+    return "stringValue";
+  } else if (isUnionType(type)) {
+    const types = parseUnionType(type);
+    return pipe(map(typeToValue), reject(isNil), nth(0))(types);
+  }
+
+  return undefined;
```

Comment:
Return nothing?
https://github.com/sindresorhus/meta/issues/7

Reply:
Definitely possible, the return makes it more explicit though. I'm happy to remove it if you guys feel it's useless.

Reply:
https://github.com/sindresorhus/meta/issues/7#issuecomment-480599355 ❤️